src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -75,22 +75,18 @@ const SearchResultsPage = (props, context) => {
         <div className="nypl-page-header">
           <div className="nypl-full-width-wrapper">
             {breadcrumbs}
-            <h1 aria-label={h1Label}>
+            <h2 aria-label={h1Label}>
```

Comment:
Nice. Good catch. I will change this in the v3 item page branch, which I cut off of this one.